### ISSUE TITLE
h5 to pb in Tensorflow 2.x

### ISSUE THREAD

**Issue by sctrueew** [amir-abdi/keras_to_tensorflow#118]
Hi,

I have trained a Mask-RCNN with TensorFlow 2 and, How can I convert h5 to pb in  TensorFlow 2?

Thanks
